docs/zh/community/people.md

```diff
@@ -0,0 +1,33 @@
+# Layotto社区成员
+注：所有排名不分先后
+## PMC Member
+@seeflood @zhenjunMa @wenxuwan @MoonShining
+
+## Reviewer
+邀请中，欢迎感兴趣的Contributor联系@seeflood 加入社区、成为Reviewer
+
+|  模块   | Reviewer |
+|  ----  | --- |
+| Layotto组件  | @stulzq @ZLBer  |
+| Layotto sdk  | @stulzq @ZLBer  |
+| API定义  | @seeflood |
+| Layotto Runtime  | @seeflood @zhenjunMa |
+| WASM  | @seeflood @zhenjunMa |
+| RPC相关  | @MoonShining |
+
+
+## Member
+邀请中，欢迎感兴趣的同学联系@seeflood 加入社区
+![img.png](../../img/community/img.png)
+
+以及：
+
+![img.png](img.png)
```

Comment:
@seeflood @zhenjunMa 

Looks like this link is wrong, it won't display.

![image](https://user-images.githubusercontent.com/837658/135246665-ef64acf1-cbc7-4776-bfdc-41fe881abde6.png)

Reply:
Ok, it will be fixed in the #234 PR